cli/spec/kontena/cli/stacks/list_command_spec.rb

```diff
@@ -60,4 +61,77 @@
       end
     end
   end
+
+  context "command output layout" do
+    before do
+      allow(client).to receive(:get).with('grids/test-grid/stacks').and_return({
+        'stacks' => []
+      })
+    end
+
+    it "renders NAME as 1st column" do
+      expect{subject.run([])}.to output_lines [/^NAME/]
+    end
+
+    it "renders STACK as 2nd column" do
+      expect{subject.run([])}.to output_lines [/STACK/]
+    end
+
+    it "renders SERVICES_COUNT as 3rd column" do
+      expect{subject.run([])}.to output_lines [/SERVICES_COUNT/]
+    end
+
+    it "renders STATE as 4th column" do
+      expect{subject.run([])}.to output_lines [/STATE/]
+    end
+
+    it "renders PORTS as 5th column" do
+      expect{subject.run([])}.to output_lines [/PORTS/]
+    end
+
+    it "renders LABELS as 6th column" do
+      expect{subject.run([])}.to output_lines [/LABELS$/]
+    end
+  end
+
+  context "with stack and labels" do
+    before do
+      allow(client).to receive(:get).with('grids/test-grid/stacks').and_return({
+        'stacks' => [{
+          'name' => 'stack-a',
+          'stack' => 'foo/stack-a',
+          'services' => [],
+          'variables' => { 'foo' => 'bar' },
+          'children' => [],
+          'labels' => ['fqdn=about.me']
+        }]
+      })
+    end
+
+    it "outputs stack labels" do
+      expect{subject.run([])}.to output_table [
+        ["\e[2m\u229D\e[0m", 'stack-a', 'foo/stack-a:', '0', 'fqdn=about.me'],
```

Comment:
Maybe these icon sequences could be replaced with something.

I think the output helper understands rspec matchers such as:

```ruby
[anything, 'stack-a', .. ]
[an_instance_of(String), 'stack-a', .. ]
```

Reply:
✅ RESOLVED